config/boards/olimex-teres-a64.conf

```diff
@@ -0,0 +1,36 @@
+# OLIMEX Teres-A64
+# Maintainer: Jacob "Kreyren" Hrbek <kreyren@fsfe.org>
+
+# Optimized for Mainboard Rev.C
+
+BOARD_NAME="OLIMEX Teres A64" # Allwinner A64 quad core 2GB SoC Wi-Fi/BT
+
+# Specified in https://linux-sunxi.org/Allwinner_SoC_Family
+BOARDFAMILY="sun50iw1"
+
+# Has custom default configuration of the u-boot that should be used -- https://linux-sunxi.org/Olimex_Teres-A64#Mainline_U-Boot
```

Comment:
All boards have custom u-boot configs. Remove.

Reply:
I prefer relevant references in my code so that it's easier to navigate while debuging something, do you mind?

Reply:
No reply for 4 days, considering stale

Reply:
So you never came to the idea that other people have some sort of **real life** besides sitting in front of a monitor?
Putting pressure is considered abusive behavior.

Reply:
Hm O.o I am used to doing 3 days or solved, but i don't mind adapting if you want me to wait longer for conversations to be answered